tools/automation/cache/build_new_cache.sh

```diff
@@ -1,7 +1,7 @@
 #!/bin/sh -ex
 bundle install
 rm db/modules_metadata_base.json
-git ls-files modules/ -z | xargs -0 -n1 -I{} -- git log -1 --format="%ai {}" {} | while read -r udate utime utz ufile ; do
+git ls-files modules/ -z | xargs -0 -n1 -P 8 -I{} -- git log -1 --format="%ai {}" {} | while read -r udate utime utz ufile ; do
```

Comment:
Environment:

```
docker run --rm=true --tty \
  --volume=`pwd`:/r7-source \
  -it \
   --workdir=/r7-source metasploitframework/metasploit-framework:latest \
  /bin/sh

```

Confirming xargs supports `-P`

```
BusyBox v1.36.1 (2023-11-06 11:32:24 UTC) multi-call binary.

Usage: xargs [OPTIONS] [PROG ARGS]

Run PROG on every item given by stdin

	-0	NUL terminated input
	-a FILE	Read from FILE instead of stdin
	-o	Reopen stdin as /dev/tty
	-r	Don't run command if input is empty
	-t	Print the command on stderr before execution
	-p	Ask user whether to run each command
	-E STR,-e[STR]	STR stops input processing
	-I STR	Replace STR within PROG ARGS with input line
	-n N	Pass no more than N args to PROG
	-s N	Pass command line of no more than N bytes
	-P N	Run up to N PROGs in parallel <----------------------
	-x	Exit if size is exceeded
```

Reply:
Why not ```-P `nproc` ``` instead of hardcoding `8`?